### DISCUSSION TITLE
B站弹幕显示不清晰:not(.bili-dm,.bili-dm *)部分浏览器/系统不生效

### DISCUSSION THREAD

**Discussion by AmazingMax** [F9y4ng/GreasyFork-Scripts#353] [问答专区(Question / Answer)] [answered]
台式机Win11系统，Vivaldi浏览器最新版6.8.3381.46 ，篡改猴测试版，在 需要渲染的网页元素: 文本域 追加 :not(.bili-dm,.bili-dm *) 后，不生效，弹幕依然看不清，重置保存刷新都尝试了，禁用其他所有插件以及脚本也都测试了。（脚本的其他功能比如我设置排除渲染B站是生效的）
但是，在其他相同条件下换成360极速浏览器就正常了（其他浏览器没有再测试了）；
以及，再换成Win10笔记本+其他相同条件，2个浏览器也都正常生效。
希望作者能帮忙排查一下是哪一环有问题。
![PixPin_2024-06-26_12-22-17](https://github.com/F9y4ng/GreasyFork-Scripts/assets/99153663/9d9db78b-762e-428f-8350-e24a0c5489d0)

**Comment by F9y4ng** [comment]
建议你在win11下使用其他同步更新的BLINK内核浏览器如Edge或Chrome试一试，360的内核版本较低不具有参考性。

如果其他BLINK内核浏览器正常，那就是Vivaldi上的某些设置或浏览器本身造成的问题。

PS. 检查一下浏览器缓存，清空所有缓存后重启浏览器，在扩展管理器中重启TM扩展，然后重试一下。因有极小的概率会发生浏览器扩展缓存错误，造成缓存数据无法更新生效的问题。

**Reply by AmazingMax** [reply, marked as answer]
最后还是卸载油猴重装解决了。（遇事不决重启/重装，注意备份脚本）
清浏览器缓存也不行，试了最新的Edge和Chrome都能正常生效。

**Reply by F9y4ng** [reply]
> （遇事不决重启/重装，注意备份脚本）

That's right.

**Comment by syy2153750** [comment]
遇到一样的问题了，用的百分浏览器，某次更新突然不行了，重装也没用

**Reply by F9y4ng** [reply]
> 用的百分浏览器

如果`disabled DirectWrite`，则在某个更新后将不在被提供支持。 [Wiki](https://github.com/F9y4ng/GreasyFork-Scripts/wiki#-%E9%83%91%E9%87%8D%E5%A3%B0%E6%98%8E) 中已有相应的提示。

**Comment by sy0sy0** [comment]
遇到同样的问题，前几天突然失效，chrome版本126.0.6478.183，尝试重装油猴、清理缓存也不管用

**Comment by F9y4ng** [comment]
@syy2153750 @sy0sy0 参考 #360
